ui/packages/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js

```diff
@@ -136,8 +139,30 @@ module.exports = Marionette.LayoutView.extend({
         }));
     },
     focus: function () {
+        this.autoLoadDefaultIfSet();
```

Comment:
❓ Why are we calling this specifically when `focus()` is called?

Reply:
This is the only function I could get to auto-call when the dropdown opens. I'm open to suggestions.